sql-reportingservices-two-machines/azuredeploy.json

```diff
@@ -0,0 +1,553 @@
+      "type": "string",
+      "defaultValue": "Enterprise",
+      "allowedValues": [
+        "Enterprise",
+        "Evaluation"
+      ],
+      "metadata": {
+        "description": "The SQL SKU, for SQL CTP Versions the only available is evaluation"  
+      }
+    },
+    "vmPrefix": {
+      "type": "string",
+      "minLength": 1,
+      "defaultValue": "SSRS",
+      "metadata": {
+        "description": "Prefix of the Virtual Machines (Prefix-RS, Prefix-Catalog)"
+      }
+    },
+    "vmSize": {
+      "type": "string",
+      "defaultValue": "Standard_D3",
+      "metadata": {
+        "description": "The size of the VM Created"
+      }
+    },
+    "vhdStorageType": {
+      "type": "string",
+      "allowedValues": [
+        "Standard_LRS",
+        "Premium_LRS"
+      ],
+      "defaultValue": "Standard_LRS",
+      "metadata": {
+        "description": "VHD storage type"
+      }
+    }
+  },
+  "variables": {
+    "sqlImagePublisher": "MicrosoftSQLServer",
+    "sqlImageOffer": "[parameters('sqlImageVersion')]",
+    "sqlImageSku": "[parameters('sqlImageSKU')]",
+    "OSDiskName": "osdiskforwindowssimple",
+    "dataDiskName": "dataDisk",
+    "nicName": "SSRSNic",
+    "addressPrefix": "10.0.0.0/16",
+    "subnetName": "Subnet",
+    "subnetPrefix": "10.0.0.0/24",
+    "vhdStorageType": "[parameters('vhdStorageType')]",
+    "publicIPAddressName": "myPublicIP",
+    "publicIPAddressType": "Dynamic",
+    "vhdStorageContainerName": "vhds",
+    "vmNameRs": "[concat(parameters('vmPrefix'), '-RS')]",
+    "vmSizeRs": "[parameters('vmSize')]",
+    "vmImageOfferRs": "[variables('sqlImageOffer')]",
+    "vmImageSkuRs": "[variables('sqlImageSku')]",
+    "vmSizeDataSource": "[parameters('vmSize')]",
+    "vmNameDataSource": "[concat(parameters('vmPrefix'), '-DataSource')]",
+    "vmSizeCatalog": "[parameters('vmSize')]",
+    "vmNameCatalog": "[concat(parameters('vmPrefix'), '-Catalog')]",
+    "virtualNetworkName": "SSRSVNET",
+    "vnetId": "[resourceId('Microsoft.Network/virtualNetworks', variables('virtualNetworkName'))]",
+    "subnetRef": "[concat(variables('vnetId'), '/subnets/', variables('subnetName'))]",
+    "vhdStorageNameRs": "[concat('vhdrs', uniqueString(resourceGroup().id))]",
+    "vhdStorageNameDataSource": "[concat('vhdds', uniqueString(resourceGroup().id))]",
+    "vhdStorageNameCatalog": "[concat('vhdcatalog', uniqueString(resourceGroup().id))]",
+    "catalogPublicIpId": "[concat('Microsoft.Network/publicIPAddresses/',concat(variables('publicIPAddressName'), 0))]",
+    "dnsName": "[parameters('dnsNameForPublicIP')]",
+    "sqlConfigurationConfigurationFunction": "PrepareSqlServer.ps1\\PrepareSqlServer",
+    "sqlSAConfigurationConfigurationFunction": "PrepareSqlServerSa.ps1\\PrepareSqlServerSa",
+    "rsConfigurationConfigurationFunction": "PrepareRsServer.ps1\\PrepareSSRSServer",
+    "sqlModuleUrl": "https://raw.githubusercontent.com/jtarquino/RsArm/master/RSARM/DSC/PrepareSqlServer.ps1.zip",
```

Comment:
This should point to the module in this repo. You can put Azure link in here and the CI will replace this to your repo's location for test. See https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#travis-ci